src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -138,19 +138,13 @@ public override IEnumerable<DownloadClientItem> GetItems()
                     DownloadClientInfo = DownloadClientItemClientInfo.FromDownloadClient(this),
                     RemainingSize = (long)(torrent.Size * (1.0 - torrent.Progress)),
                     RemainingTime = GetRemainingTime(torrent),
-                    SeedRatio = torrent.Ratio,
-                    OutputPath = _remotePathMappingService.RemapRemoteToLocal(Settings.Host, new OsPath(torrent.SavePath)),
+                    SeedRatio = torrent.Ratio
                 };
 
                 // Avoid removing torrents that haven't reached the global max ratio.
                 // Removal also requires the torrent to be paused, in case a higher max ratio was set on the torrent itself (which is not exposed by the api).
                 item.CanMoveFiles = item.CanBeRemoved = (torrent.State == "pausedUP" && HasReachedSeedLimit(torrent, config));
 
```

Comment:
We should cache the SavePath provided by GetItems somewhere, because if SavePath changes then the user moved the files/output dir. Which means you have to force it to refresh the Deferred Torrent details.
Normally this was handled implicitly.

Alternatively we can have a timer that allows PopulateTorrentDetails to refresh it every 5-15 minutes or so.